datavalidation.go

```diff
@@ -261,6 +261,29 @@ func (f *File) AddDataValidation(sheet string, dv *DataValidation) error {
 	return err
 }
 
+// GetDataValidation retrieves all comments and returns a map of worksheet name to
+// the worksheet data validations.
+func (f *File) GetDataValidation() (map[string][]*DataValidation, error) {
+	resultDVs := map[string][]*DataValidation{}
+	for sheet := range f.sheetMap {
+		ws, err := f.workSheetReader(sheet)
+		if err != nil {
+			return nil, err
+		}
+
+		// No data validaion on this sheet
+		if ws.DataValidations == nil {
+			return resultDVs, nil
+		}
+
+		for _, item := range ws.DataValidations.DataValidation {
+			dv := *item
```

Comment:
The variable `item` is a pointer of the data validation, this code can be simplified by appending it to the result map directly.